refactor(appkit): migrate PageUnauthorized component to AppKit (#1595)
commercetools/merchant-center-application-kit: packages/application-components/src/components/page-unauthorized/README.md

# PageUnauthorized

## Description

> The PageUnauthorized component can be used for informing user they don't have permissions for certain resources in Merchant Center applications.

Reviewer: ```suggestion
The PageUnauthorized component can be used to inform a user that certain permissions are lacking for views of a respective Merchant Center application.
```

Author: :+1:
